x/ethbridge/client/cli/query.go

```diff
@@ -14,27 +15,31 @@ import (
 // GetCmdGetEthBridgeProphecy queries information about a specific prophecy
 func GetCmdGetEthBridgeProphecy(queryRoute string, cdc *codec.Codec) *cobra.Command {
 	return &cobra.Command{
-		Use:   "prophecy [ethereum-chain-id] [bridge-contract] [nonce] [symbol] [token-contract] [ethereum-sender]",
+		Use: `prophecy [bridge-contract] [nonce] [symbol] [ethereum-sender]
+		--ethereum-chain-id [ethereum-chain-id] --token-contract-address [token-contract-address]`,
 		Short: "Query prophecy",
-		Args:  cobra.ExactArgs(6),
+		Args:  cobra.ExactArgs(4),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			cliCtx := context.NewCLIContext().WithCodec(cdc)
 
-			ethereumChainID, err := strconv.Atoi(args[0])
+			ethereumChainIDString := viper.GetString(types.FlagEthereumChainID)
+			ethereumChainID, err := strconv.Atoi(ethereumChainIDString)
```

Comment:
does this throw an error if the input is not a number? or does it also accept things like "one"? Could be bad if it accepted real text representation of numbers, someone thinks the chain id is "one" but it uses "1" instead? or is this just for internal accounting... maybe it doesn't matter?

Reply:
Throws an error if the input is not a number. Examples:
- `--ethereum-chain-id=3`: successful request
- `--ethereum-chain-id=three`: "ERROR: strconv.Atoi: parsing "three": invalid syntax"
- `--ethereum-chain-id=ropsten`: "ERROR: strconv.Atoi: parsing "ropsten": invalid syntax"

Reply:
nice : )